enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/APIFactory.java

```diff
@@ -71,13 +75,19 @@ public void addApis(List<Api> apis) {
                 webSocketAPI.init(api);
                 String apiKey = getApiKey(webSocketAPI);
                 newApis.put(apiKey, webSocketAPI);
+            } else if (APIConstants.ApiType.GRAPHQL.equals(api.getApiType())) {
```

Comment:
Shall we go with !GraphQL first assuming Rest APIs would we used more often?

Reply:
it doesn't make much difference, does it? This format is fine imho as it is extendable if other API types come.

Reply:
Raised because of a comment I got during a code review. I was asked keep the most frequently used condition at the very top. But I agree with your point where there is another type and a RestAPI is fine. +1